src/index.js

```diff
@@ -1,14 +1,20 @@
-require('./app')
-require('./api')
-require('./babbage')
-require('./chart')
-require('./crosstab')
-require('./facts')
-require('./pager')
-require('./panel')
-require('./sankey')
-require('./treemap')
-require('./util')
-require('./workspace')
+//require('./app')
```

Comment:
I can delete all old components if we will use our fork only. New components are a library but old code base is an application. So new code is not equal to old one.